### PR TITLE
add contact section

### PR DESCRIPTION
**Pull request by iamtodor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,13 +116,13 @@ The software is provided as is under no guarantee.
 I will update both the repo and this readme as I go during following days (date of writing this is 26th of February 2022, third day into russian invasion into Ukraine)
 
 Synflood implementation is taken from https://github.com/bilalcaliskan/syn-flood and slightly patched. I couldn't just import the package as all the functionality code was in an internal package preventing import into other modules. Will figure it out better later (sorry to the owner).
-## How to install
+# How to install
```
Comment (Arriven) [inline README.md:119]:
```suggestion
## How to install
```

This is the proper markdown formatting according to my linter (there can be only one top level heading in a document)

Comment (Arriven) [reply README.md:119]: I'll add and apply other suggestions based on that (just found out it's possible to do this in github)

File: README.md [hunk 2]
```diff
@@ -116,13 +116,13 @@ The software is provided as is under no guarantee.
...
 
-### binary install
+## binary install
```
Comment (Arriven) [inline README.md:121]:
```suggestion
### binary install
```

File: README.md [hunk 3]
```diff
@@ -116,13 +116,13 @@ The software is provided as is under no guarantee.
...
 
 go to releases page and install latest version for your os
 
-### go install
+## go install
```
Comment (Arriven) [inline README.md:125]:
```suggestion
### go install
```

File: README.md [hunk 4]
```diff
@@ -131,7 +131,7 @@ go install github.com/Arriven/db1000n@latest
 ~/go/bin/db1000n
 ```
 
-### docker install with openvpn
+## docker install with openvpn
```
Comment (Arriven) [inline README.md:134]:
```suggestion
### docker install with openvpn
```

File: README.md [hunk 5]
```diff
@@ -154,11 +154,11 @@ if you want to to use VPN from within docker container:
 ./run.sh
 ```
 
-### kubernetes install
+## kubernetes install
```
Comment (Arriven) [inline README.md:157]:
```suggestion
### kubernetes install
```

File: README.md [hunk 6]
```diff
@@ -154,11 +154,11 @@ if you want to to use VPN from within docker container:
...
 
 for experienced users, see instructions in [helm/](helm/)
 
-### shell install
+## shell install
```
Comment (Arriven) [inline README.md:161]:
```suggestion
### shell install
```

File: README.md [hunk 7]
```diff
@@ -172,7 +172,7 @@ the command above will detect the os and architecture, dowload the archive, vali
 ./db1000n
 ```
 
-### k8s run
+## k8s run
```
Comment (Arriven) [inline README.md:175]:
```suggestion
### k8s run
```

File: README.md [hunk 8]
```diff
@@ -192,9 +192,9 @@ Scale it, if you have resources
 $ kubectl scale deployment/db1000n --replicas=10 -n db1000n
 ```
 
-## Configuration
+# Configuration
```
Comment (Arriven) [inline README.md:195]:
```suggestion
## Configuration
```

File: README.md [hunk 9]
```diff
@@ -192,9 +192,9 @@ Scale it, if you have resources
...
 
-### Commandline reference
+## Commandline reference
```
Comment (Arriven) [inline README.md:197]:
```suggestion
### Commandline reference
```

File: README.md [hunk 10]
```diff
@@ -211,7 +211,7 @@ Usage of /tmp/go-build781992389/b001/exe/main:
         refresh timeout for updating the config (default 1m0s)
 ```
 
-### Config file reference
+## Config file reference
```
Comment (Arriven) [inline README.md:214]:
```suggestion
### Config file reference
```

**Review by Arriven — COMMENTED** [review]
I'll bring back the old formatting. It's not as good looking but it won't budge me that linter is bitching about it

**Comment by Arriven** [comment]
Ah, ok, it is only possible to commit suggestions if you are have write permissions to the fork. I thought it's a bit smarter than that

File: README.md (hunk 1 above)
Comment (iamtodor) [reply README.md]: @Arriven Perhaps I am using markdown a bit wrongly, but I see no point having only one top-level heading. I used to separate different categories by top-level heading and then go down accordingly

**Comment by iamtodor** [comment]
@Arriven I updated the contact section, the data was taken from the channel info

**Comment by Arriven** [comment]
There were conflicts due to #128 so I applied the patch manually

**Pull request closed without merging**